frontend/src/components/homepage/messages.js

```diff
@@ -65,12 +65,12 @@ export default defineMessages({
   mappingFlowTitle: {
     id: 'home.mappingFlow.title',
     defaultMessage:
-      "Each year, disasters around the world kill nearly {number} and affect or displace 200 million people. We're on a mission to change that. Here's how.",
+      "Each year, disasters around the world kill nearly 100,000 and affect or displace 200 million people. HOT is on a mission to change this.",
```

Comment:
we need the {number} in order to have it formatted on languages that uses dot instead of commas to mark the thousands